1-js/02-first-steps/08-operators/4-fix-prompt/task.md

```diff
@@ -1,18 +1,18 @@
-importance: 5
+importância: 5
 
 ---
 
```

Comment:
O nome da chave `inportance` no começo dos arquivos `task.md` deve ser mantido.

> Não traduzir as chaves de identificação dos **metadados**, pois não fazem parte do conteúdo do artigo, mas sim para fins do projeto ["server"](https://github.com/javascript-tutorial/server) que construirá o site "https://javascript.info", sendo assim, o conteúdo acima  de `---` provavelmente é utilizado pela plataforma para alguma operação ou estruturação necessária.

Por gentileza, retroceder a alteração da chave `importance` em todos os arquivos:

1-increment-order
2-assignment-result
3-primitive-conversions-questions
4-fix-prompt

Reply:
Perfeito, @jonnathan-ls ! Muito obrigado por me avisar sobre este detalhe importante! 

Esta foi precisamente a dúvida que eu tive no momento. Quando tive esta dúvida, fui pesquisar as traduções de outras pessoas, e vi que estas outras traduções haviam também traduzido a keyword, e aí resolvi seguir o padrão ([exemplo de tradução para "importância"](https://github.com/javascript-tutorial/pt.javascript.info/blob/master/1-js/02-first-steps/05-types/1-string-quotes/task.md)).

Acabei de desfazer este erro aqui: [Undoing keyword translations for the execises](https://github.com/javascript-tutorial/pt.javascript.info/pull/247/commits/b97f71208890d8b9fd7c45228222a08aaeefbb8c).